chart/application/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: epinio-application
+description: The helm chart for epinio applications to be deployed by
+icon: https://raw.githubusercontent.com/epinio/helm-charts/main/assets/epinio.png
+home: https://github.com/epinio/epinio
+type: application
+version: 0.1.8
+appVersion: "1.16.0"
```

Comment:
Should these versions match something? What is `appVersion` in this case?

Reply:
e69a351c48a6bd27093811423cadf4306bdc320c App version now matches version. Keep it matched in the future.